apps/condo/domains/acquiring/schema/SumPaymentsService.test.js

```diff
@@ -0,0 +1,140 @@
+/**
+ * Generated by `createservice acquiring.SumPaymentsService`
+ */
+
+const { makeClient } = require('@condo/keystone/test.utils')
+
+const { makePayer, createTestPayment } = require('../utils/testSchema')
+const { SUM_PAYMENTS_MUTATION } = require('../gql')
+const Big = require('big.js')
+const { makeClientWithSupportUser, makeClientWithNewRegisteredAndLoggedInUser } = require('../../user/utils/testSchema')
+const { createTestOrganizationEmployeeRole, createTestOrganizationEmployee } = require('../../organization/utils/testSchema')
+ 
+describe('SumPaymentsService', () => {
+    test('admin: sum one payment', async () => {
+        const { admin, billingReceipts, acquiringContext, organization } = await makePayer()
+        const [payment] = await createTestPayment(admin, organization, billingReceipts[0], acquiringContext)
+        const where = { organization: { id: organization.id } }
+        const {
+            data: {
+                result: {
+                    sum,
+                },
+            },
+        } = await admin.query(SUM_PAYMENTS_MUTATION, { where: where })
```

Comment:
There is a way to invoke custom queries by test utils. Check out AllResidentBillingReceipts tests

Reply:
AllResidentBillingReceipts tests invokes custom mutation by test utils function with client.mutation inside